pages/linux/systemctl.md

```diff
@@ -11,26 +11,26 @@
 
 `systemctl --failed`
 
-- Start/Stop/Restart/Reload a service:
+- Start/Stop/Restart/Reload/Show the status a service:
 
-`systemctl {{start|stop|restart|reload}} {{unit}}`
-
-- Show the status of a unit:
-
-`systemctl status {{unit}}`
+`systemctl {{start|stop|restart|reload|status}} {{unit}}`
 
 - Enable/Disable a unit to be started on bootup:
 
 `systemctl {{enable|disable}} {{unit}}`
 
-- Mask/Unmask a unit to prevent enablement and manual activation:
+- Reload systemd, scan for new or changed units:
+
+`systemctl daemon-reload`
 
-`systemctl {{mask|unmask}} {{unit}}`
+- Check if a unit is active/enabled/failed:
 
-- Reload systemd, scanning for new or changed units:
+`systemctl {{is-active|is-enabled|is-failed|}} {{unit}}`
 
-`systemctl daemon-reload`
+- List all service/socket/automount units filtering by running/failed state:
+
+`systemctl list-units --type={{service|socket|automount}} --state={{failed|running}}`
 
-- Check if a unit is enabled:
+- Show the source unit files:
```

Comment:
```suggestion
- Show the contents of a unit file:
```
I feel like this wording might be more self-explanatory

Reply:
thanks, I just updated it